controllers/cities/cities.service.js

```diff
@@ -0,0 +1,19 @@
+import { cities } from '../../models';
+
+class CitiesService {
+  getCities = () => cities.find();
+
+  getRandomCityManually = () => {
```

Comment:
Why to define this function in such way if it was already defined below?

Reply:
Because as I understood task required 2 different implementations using built-in tools and manually